cwl_schema.json

```diff
@@ -0,0 +1,6270 @@
+{
+    "$ref": "#/definitions/CWLFile",
+    "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Are there any advantages to producing a schema using the newer drafts of the JSON schema spec?

Reply:
Potentially, JetBrains only seems to support version 7 though from the drop down menus, but I don't know if they actually support more recent versions (2019 and 2020). From the JSON Schema Store, GitHub Actions schema is also on version 7 still